projects/js-packages/charts/src/components/pie-semi-circle-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,458 @@
+
+<Source
+	language="jsx"
+	code={ `<PieSemiCircleChart
+		data={ data }
+		showLegend={ true }
+		legendShape="square" // Options: 'circle', 'square', 'line'
+	/>` }
+/>
+
+## Error Handling
+
+### Data Validation
+
+The chart includes comprehensive data validation with user-friendly error messages:
+
+<Canvas of={ PieSemiCircleChartStories.ErrorStates } />
+
+**Validation Rules:**
+- Data array cannot be empty
+- Values and percentages must be non-negative
+- Total percentage must be greater than 0 (unlike full pie charts, doesn't need to equal 100)
+
+**Error States Handled:**
+- **Empty Data**: Displays "No data available"
+- **Negative Values**: Shows "Invalid data: Negative values are not allowed"
+- **Zero Total**: Displays "Invalid percentage total: Must be greater than 0"
+
+### Single Data Point
+
+The chart gracefully handles single data points, rendering a complete semi-circle:
+
+<Source
+	language="jsx"
+	code={ `const singlePointData = [
+		{
+			label: 'Complete',
+			value: 100,
+			percentage: 100,
+		},
+	];
+
+	<PieSemiCircleChart
+		data={ singlePointData }
+		width={ 400 }
+		label="Single Category"
+	/>` }
+/>
+
+## Advanced Features
+
+### Value Display Formatting
+
+Use the `valueDisplay` property to show formatted values in tooltips and legends:
+
+<Source
+	language="jsx"
+	code={ `const formattedData = [
+		{
+			label: 'Users',
+			value: 15000,
+			valueDisplay: '15K users', // Custom formatted display
+			percentage: 60,
+		},
+		{
+			label: 'Revenue',
+			value: 10000,
+			valueDisplay: '$10K', // Currency formatting
+			percentage: 40,
+		},
+	];
+
+	<PieSemiCircleChart
+		data={ formattedData }
+		withTooltips={ true }
+		showLegend={ true }
+	/>` }
+/>
+
+### Chart Integration
+
+The chart integrates with the chart context system for consistent behavior across chart types:
+
+- Automatic chart ID generation for accessibility
+- Theme consistency across multiple charts
+- Legend item registration for cross-chart interactions
+
+## Accessibility
+
+### Screen Reader Support
+
+The chart provides comprehensive screen reader support:
+- ARIA labels describe the chart purpose and data
+- Individual segments are announced with labels and values
+- Error states are clearly communicated
+- Legend items are properly associated with chart segments
+
+## Browser Compatibility
+
+### Modern Browser Support
+
+The chart uses modern SVG and CSS features with broad browser support:
+- **Chrome/Edge**: Full support for all features
+- **Firefox**: Complete functionality including animations
+- **Safari**: Full support with minor rendering optimizations
+
+### Performance Considerations
+
+- **Rendering**: Optimized SVG rendering for smooth interactions
+- **Memory**: Efficient re-rendering with React hooks optimization
+- **Animations**: CSS-based transitions for hover states
+
+## API Reference
+
+### PieSemiCircleChart
+
+Main component for rendering semi-circular pie charts.
+
+**Props:**
```

Comment:
This is missing a number of props including height, margin, the pointer event handlers, etc